### PR TITLE
Extend transfer validators

### PR DESCRIPTION
**Pull request by cehali**
# Summary

### CODE REVIEW

File: universal-login-sdk/lib/core/services/validations/RecipientValidator.ts [hunk 1]
```diff
@@ -1,28 +1,37 @@
 import {Validator, TransferErrors} from './Validator';
 import {utils} from 'ethers';
 import {isValidEnsName, TransferDetails} from '@universal-login/commons';
+import UniversalLoginSDK from '../../../api/sdk';
+import {AddressZero} from 'ethers/constants';
 
-function isProperAddress(recipient: string): boolean {
+const isProperAddress = (recipient: string): boolean => {
```
Comment (xitronix) [inline universal-login-sdk/lib/core/services/validations/RecipientValidator.ts]: Merge with isProperAddress in commons

File: universal-login-wallet/test/integration/ui/Transfer.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ describe('UI: Transfer', () => {
       : '0x0';
   });
 
-  it('send ETH => invalid ensName', async () => {
+  xit('send ETH => invalid ensName', async () => {
```
Comment (xitronix) [inline universal-login-wallet/test/integration/ui/Transfer.tsx]: remove xit

File: universal-login-sdk/lib/core/services/validations/RecipientValidator.ts (hunk 1 above)
Comment (cehali) [reply universal-login-sdk/lib/core/services/validations/RecipientValidator.ts]: isProperAddress in commons looks like this:
```ts
export const isProperAddress = (address: string) => {
  return !!address.match(/^0x[0-9A-Fa-f]{40}$/);
};
```
And here we use ethers to define if address is valid.

File: universal-login-wallet/test/integration/ui/Transfer.tsx (hunk 2 above)
Comment (cehali) [reply universal-login-wallet/test/integration/ui/Transfer.tsx]: 👍

File: universal-login-sdk/lib/core/services/validations/RecipientValidator.ts (hunk 1 above)
Comment (cehali) [reply universal-login-sdk/lib/core/services/validations/RecipientValidator.ts]: I can update isProperAddress in commons in next PR

**Pull request merged**